whyis/filters.py

```diff
@@ -139,7 +139,7 @@ def attributes(query, this):
         result['description'] = sorted(result['description'], key=lambda x: len(x['value']))
         thumbnail = this.description().value(app.NS.foaf.depiction)
         if thumbnail is not None:
-            result['thumbnail'] = thumbnail.identifier
+            result['thumbnail'] = thumbnail.value if type(thumbnail) is rdflib.Literal else thumbnail.identifier
```

Comment:
this shouldn't be needed here, although it does provide flexibility if someone misinterprets foaf:depiction.